samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/SendMessageController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using NServiceBus;
+
+[Route("api/[controller]")]
+public class SendMessageController : Controller
+{
+    IMessageSession messageSession;
+
+    #region MessageSessionInjection
+    public SendMessageController(IMessageSession messageSession)
+    {
+        this.messageSession = messageSession;
+    }
+    #endregion
+
+
+    #region MessageSessionUsage
+    [HttpGet]
+    public async Task<string> Get()
```

Comment:
I see your point, btut the Web API is really all there is here. If we don't piggyback on the resulting GET from "open this url at startup", we have to include a API exerciser which is really not the focus in this sample.